index.html

```diff
@@ -5,19 +5,20 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Mad Libs Word Game - Chingu Voyage 43 - Group 9</title>
+    <link rel="stylesheet" href="index_mobile.css" />
   </head>
-  <body>
-    <header>
+  <body class="flex">
```

Comment:
In this line you are telling the html that the `<body>` element has a class of `flex`.

Reply:
Before the code even moves on to the next line it has declared a class of `flex` to body as mentioned above.
This means that this element's css looks like this
```
  display: flex;
  flex-direction: column;
  align-items: center;
  text-align: center
```

Reply:
You probably want this effect and this is fine but there is a part of your css we left out because well it doesn't come into play yet as far as this html page cares so we move on...